ui/src/components/NodeListTable.js

```diff
@@ -420,25 +36,38 @@ const NodeListTable = (props) => {
         Header: 'Health',
         accessor: 'health',
         cellStyle: { textAlign: 'center', width: '5rem' },
-        Cell: (cellProps) => {
-          const { health } = cellProps.value;
-          return <CircleStatus status={health} />;
+        Cell: ({ value }) => {
+          return <CircleStatus status={value.health} />;
         },
-        sortType: 'health',
       },
       {
         Header: 'Name',
         accessor: 'name',
-        sortType: 'name',
+        cellStyle: { flex: 1 },
+        Cell: ({ value }) => {
+          const { name, controlPlaneIP, workloadPlaneIP } = value;
+          return (
+            <>
+              <LargeText data-cy="node_table_name_cell">{name}</LargeText>
+              <Box display={'inline-flex'} flexWrap={'wrap'}>
+                {controlPlaneIP ? <IPText>CP : {controlPlaneIP}</IPText> : null}
```

Comment:
We should try to use an "official" text component or maybe split the info in 2 additional columns.